src/monattack.cpp

```diff
@@ -514,7 +514,7 @@ bool mattack::acid_accurate(monster *z)
     proj.proj_effects.insert( "NO_DAMAGE_SCALING" );
     proj.impact.add_damage( DT_ACID, rng( 3, 5 ) );
     // Make it arbitrarily less accurate at close ranges
-    z->projectile_attack( proj, target->pos(), rng( 0, 8000 / range ) );
+    z->projectile_attack( proj, target->pos(), 8000 / range );
```

Comment:
I don't have any particular opinion on what this value should be, what do you suggest it is changed to? Probably something to do in a separate PR though.